### ISSUE TITLE
Wondering about migration best practice

### ISSUE THREAD

**Issue by eteubert** [sorentwo/oban#83]
**Here's what I did:**

First migration when setting up Oban

```
  defdelegate up, to: Oban.Migrations
  defdelegate down, to: Oban.Migrations
```

Second migration that came later

```
  def up do
    Oban.Migrations.up(version: 3)
  end

  def down do
    Oban.Migrations.down(version: 2)
  end
```

**Issue with these migrations**

Now, when setting up the project from scratch, the first migration already migrates all the way and subsequent migrations fail because they try to execute versions that are already done.

The fix for me, I think, is to change the initial migration to:

```
  def up do
    Oban.Migrations.up(version: 2)
  end

  def down do
    Oban.Migrations.down(version: 1)
  end
```

However I wonder if the documentation should be adjusted so others don't run into the same issue.

**Comment by sorentwo** [comment]
This seems to be the same issue as #69, which was then fixed/guarded against in https://github.com/sorentwo/oban/commit/12bc576869048c10d12aea2c0758b337d646dcc8 and then released in [v0.9.0](https://github.com/sorentwo/oban/blob/master/CHANGELOG.md#090--2019-09-20).

There are [property tests that verify migrating up and down](https://github.com/sorentwo/oban/blob/master/test/integration/migrating_test.exs#L88) between arbitrary versions, so this shouldn't have regressed.

Which version of Oban are you using?

**Comment by eteubert** [comment]
Oh, that project is still on 0.8.1. So simply upgrading to 0.9 or higher should solve the issue? Should have tried that first, sorry!

**Comment by sorentwo** [comment]
Upgrading to 0.9+ will definitely fix it! There are some other changes in 0.9 and 0.10 that are definitely worth the upgrade.

Glad we solved it 👍
